### ISSUE TITLE
Zuul 2 has been released

### ISSUE THREAD

**Issue by kamilgregorczyk** [spencergibb/spring-cloud-gateway-bench#4]
Hi, zuul 2.1 has been released and it offers a non blocking proxy, would you like to check it out and update the readme.md file?

**Comment by imet** [comment]
You can checkout this [repos](https://github.com/imet/spring-cloud-gateway-bench), I have add the zuul2 project for bench testing.

**Comment by LukeDemons** [comment]
Hi, @imet. I cannot find your testing result in the table. Did you just provide a method?

**Comment by imet** [comment]
@LukeDemons ，yeah, you can try the test on your side, I just provide the method and project. 

Since I didn't have the same system environment with spencergibb, so currently I didn't provide my result.

I will update my benchmark later on test env.

**Comment by gihad** [comment]
I agree that this should be compared to non-blocking async implementation to be fair.

@imet What were your findings? 

My concern is that in my tests Spring Webflux is still very slow compared to Vert.x which is what we are using for async non-blocking (approximately 10x slower). I'm hoping that eventually the guys from Reactor Project will be able to optimize it and get closer since they both use Netty.

I would think that Spring Cloud Gateway would suffer from similar limitations in performance.
